src/main/java/com/example/account/service/command/impl/CommandServiceImpl.java

```diff
@@ -17,6 +18,7 @@
 @AllArgsConstructor
 public class CommandServiceImpl implements CommandService {
 
+    private final ReactiveProducerService producerService;
```

Comment:
@amerkul what is the `ReactiveProducerService`?

Reply:
Kafka sends an event next microservice

Reply:
@amerkul got it, thanks

Reply:
@amerkul I think `ReactiveProducerService` is not the best naming since it specifies the details about the paradigm, in this case, it reactive

so, we can rename it to the just `ProducerService`, and one of the concrete implementations will contain prefix `Reactive`, indicating that implementation is based on reactive programming paradigm

WDYT?

Reply:
not a problem